src/wifi/AdafruitIO_WINC1500.h

```diff
@@ -21,29 +21,77 @@
 #include "Adafruit_MQTT.h"
 #include "Adafruit_MQTT_Client.h"
 
-// feather wifi m0
-#define WINC_CS   8
-#define WINC_IRQ  7
-#define WINC_RST  4
-#define WINC_EN   2
-
 class AdafruitIO_WINC1500 : public AdafruitIO {
 
   public:
-    AdafruitIO_WINC1500(const char *user, const char *key, const char *ssid, const char *pass);
-    ~AdafruitIO_WINC1500();
+    AdafruitIO_WINC1500(const char *user, const char *key, const char *ssid, const char *pass, int winc_cs, int winc_irq, int winc_rst, int winc_en) : AdafruitIO(user, key)
```

Comment:
make the default pins

#define WINC_CS   8
#define WINC_IRQ  7
#define WINC_RST  4
#define WINC_EN   2